test/libsolidity/SolidityNatspecJSON.cpp

```diff
@@ -1318,15 +1312,82 @@ BOOST_AUTO_TEST_CASE(user_default_inherit_variable)
 	)";
 
 	char const *natspec = R"ABCDEF({
-		"methods":
+		"methods": { "x()": { "notice": "Hello world" } }
+	})ABCDEF";
+
+	checkNatspec(sourceCode, "C", natspec, true);
+	checkNatspec(sourceCode, "D", natspec, true);
+}
+
+BOOST_AUTO_TEST_CASE(dev_explicit_inherit_variable)
```

Comment:
Did these tests have to be added via cpp tests? Couldn't they have been added as command line tests? (or something that is not cpp tests)

Reply:
I suppose anything could be a cmdline test, but this file is specifically for natspecs so it does seem to be the right place..
Whether we want/should turn the whole thing into not-cpp tests is a different question (of which I'd be in favour)

Reply:
Yea, I just wondered if some similar tests had already been "converted" into cmd line tests

Reply:
I am not sure cmd line tests should be the favored target, maybe a more specific extension in (i)soltest would be better.